app/controllers/idv/review_controller.rb

```diff
@@ -76,15 +83,16 @@ def idv_agent
       )
     end
 
-    def applicant_from_params
-      app_vars = idv_session.params.select { |key, _value| Proofer::Applicant.method_defined?(key) }
-      Proofer::Applicant.new(app_vars)
-    end
-
     def init_questions_and_profile(resolution)
       idv_session.resolution = resolution
       idv_session.question_number = 0
-      idv_session.profile_from_applicant(idv_session.applicant)
+      idv_session.profile_from_applicant(idv_session.applicant, password)
+    end
+
+    def password
+      params.require(:user)[:password]
+    rescue ActionController::ParameterMissing
+      ''
```

Comment:
the error message will be generated by the `User` model when attempting to validate the password. I.e. a missing password is treated like an empty string because `nil` is not allowed as an argument to `valid_password?`.